okd-release-pipeline/release.md

```diff
@@ -0,0 +1,1035 @@
+# Prerequisites
```

Comment:
This is a long document. Can this be condensed a b it?
The title here should be something like `# OKD Release Pipeline`